.ci/openshift-ci/cluster/deployments/daemonset_kata-installer.yaml.in

```diff
@@ -35,6 +35,18 @@ spec:
                   valueFrom:
                   fieldRef:
                    fieldPath: spec.nodeName
+                - name: QEMU_PATH
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ci.kata.sandboxedcontainers
+                      key: qemu_path
+                      optional: true
+                - name: USE_HOST_KERNEL
```

Comment:
Wouldn't this become `yes` (and if so, is that correct)?

Reply:
Hi @Jakob-Naucke , I am not sure I understood your question. The value of `USE_HOST_KERNEL` is read from a optional `configMap`; if the `configMap` is not deployed then `USE_HOST_KERNEL` will be exported to the container as an empty variable.

Reply:
ah, right. Nevermind.